From c9a335ad5c90ed465e3f99065c96f5aad0db4e65 Mon Sep 17 00:00:00 2001
From: Yu Jingzhi <jingzhi.yu@intel.com>
Date: Mon, 5 Jan 2015 22:47:58 +0800
Subject: [PATCH 204/210] ov5693:make the exposure and gain setting valid at
 the same time

For this sensor, gain effect in next frame, but exposure time effect in
next 2 frame, this patch make the exposure and gain setting valid at the same time.

Change-Id: I102b78e2e7fcba60714e8c8a0a767fc35d1913dc
Tracked-On:
https://hsdes.intel.com/home/default.html#article?id=1503900502
Signed-off-by: Yu Jingzhi <jingzhi.yu@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/34878
Reviewed-by: GuanX Wen <guanx.wen@intel.com>
Reviewed-by: Chao C Li <chao.c.li@intel.com>
Reviewed-by: Jing Tao <jing.tao@intel.com>
Approver: Wei Tao <wei.tao@intel.com>
Reviewed-by: Wei Tao <wei.tao@intel.com>
---
 drivers/external_drivers/camera/drivers/media/i2c/ov5693-ecs/ov5693.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/ov5693-ecs/ov5693.h b/drivers/external_drivers/camera/drivers/media/i2c/ov5693-ecs/ov5693.h
index 248e632..85b28a0 100755
--- a/drivers/external_drivers/camera/drivers/media/i2c/ov5693-ecs/ov5693.h
+++ b/drivers/external_drivers/camera/drivers/media/i2c/ov5693-ecs/ov5693.h
@@ -317,7 +317,7 @@ static struct ov5693_reg const ov5693_global_setting[] = {
 	{OV5693_8BIT, 0x3405, 0x00},
 	{OV5693_8BIT, 0x3406, 0x01},
 	{OV5693_8BIT, 0x3500, 0x00},
-	{OV5693_8BIT, 0x3503, 0x17},
+	{OV5693_8BIT, 0x3503, 0x07},
 	{OV5693_8BIT, 0x3504, 0x00},
 	{OV5693_8BIT, 0x3505, 0x00},
 	{OV5693_8BIT, 0x3506, 0x00},
-- 
1.9.1

